jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdeSpatMetadata.java

```diff
@@ -0,0 +1,35 @@
+package us.dot.its.jpo.ode.model;
+
+public class OdeSpatMetadata extends OdeLogMetadata {
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = -5361008186032548625L;
+
+	public enum SpatSource {
```

Comment:
@dan-du-car The way they defined this is confusing.  Seems like this enumeration is being used to mark when the message is received or being sent.  Check with Tony or Brandon if this applies to Spat messages also.

Reply:
i think this enum would be for bsm source (EV/RV meaning local vehicle broadcast vs remove vehicle broadcast), for SPaT i think the source would be RSU, V2X Hub, and maybe MMITSS

Reply:
okay, update to RSU, V2X, MMITSS, unknown?

Reply:
that make sense

Reply:
updated https://github.com/usdot-jpo-ode/jpo-ode/pull/414/commits/8162dc2ad2d02e6ed502768f7c5a2ab8e1f31463